changelog/front-commerce-3.9-2.34.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: "Release: 3.9 & 2.34"
+authors: pierre
+date: 2024-11-21T11:11
+hide_table_of_contents: true
+---
+
+## Consent Mode V2, Consistent Price Display, Gezy advanced capabilities and Extension Features API
+
+We know you're likely busy preparing for the Black Friday and the holiday
+season, so do we! 🎄 \
+For the last release of the year, we managed to deliver a handful of new
+features and improvements in Front-Commerce 3.9 and 2.34.
+
+Here's a quick overview of the key highlights in this release:
+
+- **Google Consent Mode V2**: Enhanced analytics compliance with granular
+  consent options.
+- **Consistent Price Display**: New `<PriceVariant />` component for uniform
+  price presentation across the theme.
+- **Gezy Enhancements**: Advanced capabilities including facet filters, guest
+  cart support, company accounts, advanced shipping and payment methods.
+- **X-Ray Mode for Gezy**: Streamlined content management with the Magic Button
+  for Gezy merchants.
+- **Extension Features API**: New API for cleaner and more maintainable theme
+  extensions.
+
+Let's explore the full details!
+
+<!-- truncate -->
+
+## Google Consent Mode V2 for analytics services
+
+We've enhanced our analytics capabilities to support
+[Consent Mode V2 (for European Economic Area compliance)](https://support.google.com/tagmanager/answer/13695607),
+**in both Front-Commerce 2.x and 3.x versions**. This update brings more
+granular control over user consent for analytics services, ensuring compliance
+with privacy regulations while maintaining effective data collection.
+
+[![Consent Mode V2 compliance (with the Consent Mode Inspector Chrome extension)](./assets/3.9/consent-mode-v2.png)](./assets/3.9/consent-mode-v2.png)
+
+This change includes:
+
+- **Granular Consent Options**: Users can now provide more specific consent for
+  different aspects of data collection and usage.
+- **Google Analytics and GTM integrations**: Our Google Analytics and GTM
+  plugins now supports Consent Mode V2 out of the box.
+- **Flexible Consent Management**: The new `withConsent` adapter method enables
+  dynamic updates to consent settings as user preferences change, for custom
+  plugins.
+
+Please refer to our
+[Granular Consent Updates](/docs/3.x/guides/analytics/getting-started#granular-consent-updates)
+to learn more about implementing Consent Mode V2 in your Front-Commerce project.
+For existing projects, follow our
```

Comment:
**note to self:** it would be good to have a little "debug" example in the guide of using the chrome extension "Consent Mode Inspector Chrome"

Reply:
That's what I thought also yesterday!

Could you do it please?